WebApp.Dockerfile

```diff
@@ -10,7 +10,7 @@ COPY --chown=node:node ./static/ ./static
 WORKDIR /home/node/app/frontend
 RUN npm run build
   
-FROM python:3.9.7-alpine3.14  
+FROM python:3.12 
```

Comment:
python 3.12 is in prerelease still, I would only go up as far as 3.11 in production code:

https://www.python.org/downloads/